comms/lighthouse/src/peers/peerMessagesHandler.ts

```diff
@@ -16,23 +16,10 @@ export function defaultPeerMessagesHandler({ peersService, archipelagoService, l
   }
 
   function handleHeartbeat(message: HeartbeatMessage, client: IClient) {
-    const { position, connectedPeerIds, parcel, optimizeNetwork, targetConnections, maxDistance } = message.payload
+    const { position, connectedPeerIds, parcel } = message.payload
     peersService().updateTopology(client.getId(), connectedPeerIds)
     peersService().updatePeerParcel(client.getId(), parcel)
     peersService().updatePeerPosition(client.getId(), position)
     archipelagoService().updatePeerPosition(client.getId(), position)
-
-    if (optimizeNetwork) {
```

Comment:
Is it ok to delete this if and everything inside?

Reply:
I think so. Now we will handle the updates of network using the archipelago directly